Core1Platform/UserInterface/01d_U2F.md

```diff
@@ -0,0 +1,55 @@
+[!!Set up U2F from the user side](../UsingCore1/01b_U2FLogin.md)   
+[!!Set up U2F from the administrator side](../AdministratingCore1/03_AdminSetupActindo.md)
+
+# U2F
+
+*Any workspace > Avator icon > Menu entry Settings > Tab U2F*
+
+![U2F](../../Assets/Screenshots/Core1Platform/ProfileSettings/U2F/U2F.png "[U2F]")
+
+**Edit U2F keys of user**
+
+- ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)   
+    Click this button to update the list of U2F keys.
+
+- ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)   
+    Click this button to display the columns bar and customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.
+
+- [x]     
+    Select the checkbox to display the editing toolbar. You cannot select multiple checkboxes at once.
+
+
+- ![Delete](../../Assets/Icons/Trash03.png "[Delete]") (Delete)      
+    Click this button to delete the U2F key. This button is only displayed, when the checkbox of an U2F key is selected.       
```

Comment:
of a U2F... (ich würde "a U2F" sagen, da U "ju" gelesen/ausgesprochen wird)

Reply:
done